spanner_graphs/magics.py

```diff
@@ -111,10 +112,12 @@ def _generate_html(query, project: str, instance: str, database: str, mock: bool
             server_content=server_content,
             app_content=app_content,
             query=query,
-            project=project,
-            instance=instance,
-            database=database,
-            mock=mock,
+            params=json.dumps({
```

Comment:
will this crash or return an unintended value if `instance`, `database`, or `mock` are unexpected values?